sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -358,8 +358,45 @@ The azure-core [reference documentation](https://azuresdkdocs.blob.core.windows.
 describes available configurations for retries, logging, transport protocols, and more.
 
 ## Troubleshooting
```

Comment:
Should we update the hardcoded westUS location to `<location>` since the service does not support this location yet?

Reply:
Can you tell me where I used westUS? I searched the README and only found westUS2 occurrence, which is supported.

Reply:
Oh I meant westUS2. I thought the service didn't support that, but if it does, it should be fine.

Reply:
There was a bug with that region last week (two weeks ago?), but it's since been fixed.